focus-ios/tools/export-strings.sh

```diff
@@ -16,23 +16,23 @@ echo "[*] Cloning mozilla-l10n/focusios-l10n"
 git clone https://github.com/mozilla-l10n/focusios-l10n.git focusios-l10n
 
 echo "[*] Cloning mozilla-mobile/LocalizationTools"
-[ -d tools/Localizations ] && rm -rf tools/Localizations
+[ -d focus-ios/tools/Localizations ] && rm -rf focus-ios/tools/Localizations
 git clone https://github.com/mozilla-mobile/LocalizationTools.git tools/Localizations
```

Comment:
This probably needs to clone in `focus-ios/tools/Localizations`?

Reply:
That's the new path being used

Reply:
You're not cloning in `focus-ios/tools/Localizations`, you're currently cloning in `tools/Localizations`. You need

```
git clone https://github.com/mozilla-mobile/LocalizationTools.git focus-ios/tools/Localizations
```

The script fails locally in exactly the same way as the GitHub workflow with the current main.

```
./focus-ios/tools/export-strings.sh: line 23: cd: focus-ios/tools/Localizations: No such file or directory
```

Reply:
oh I see, my bad, I missed that